config/retention.go

```diff
@@ -0,0 +1,7 @@
+package config
```

Comment:
Any reason why this is on its own file? Regardless it should probably be called `config/retention_policy.go`

Reply:
Why `config/retention_policy.go` since the other configuration files are named: `image.go`, `node_events.go` and so on as I understand @simster7 ?

Reply:
files is named after struct